docs/CHANGELOG.md

```diff
@@ -1592,7 +1592,7 @@ src/main/scala/file1.scala
 - No space before Decl.Type, fixes \#288 [\#328](https://github.com/scalameta/scalafmt/pull/328) ([olafurpg](https://github.com/olafurpg))
 - Fail fast instead of best-effort formatting [\#326](https://github.com/scalameta/scalafmt/pull/326) ([olafurpg](https://github.com/olafurpg))
 - Add macro benchmark. [\#320](https://github.com/scalameta/scalafmt/pull/320) ([olafurpg](https://github.com/olafurpg))
-- Correct all other mentions of "paretheses" [\#314](https://github.com/scalameta/scalafmt/pull/314) ([i-am-the-slime](https://github.com/i-am-the-slime))
+- Correct all other mentions of "parentheses" [\#314](https://github.com/scalameta/scalafmt/pull/314) ([i-am-the-slime](https://github.com/i-am-the-slime))
```

Comment:
I know what this is trying to do, but, it's actually ok to say it this way.

That said, if people prefer changelogs to be immutable (and hard to read), I understand and can trivially drop such changes.

Reply:
let's remove this modification as well. the pull request in question was in fact fixing spelling errors, specifically the misspelled word "paretheses". so there's nothing to fix, it's intentional.